src/python/pants/backend/docker/docker_build.py

```diff
@@ -71,7 +59,7 @@ async def build_docker_image(
         docker.build_image(
             field_set.image_tag,
             context.digest,
-            field_set.context_root,
+            ".",
```

Comment:
Do we need this arg to `build_image()` at all now?

Reply:
No, we could leave that as a default arg, and drop it here to make it cleaner. Will do.

Reply:
I mean can we just get rid of it entirely in the signature of `build_image`, since we only set it once, to a hard-coded value. We can just hard-code it in the process call?

Reply:
Sure :)